rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -61,6 +62,54 @@ trait PersistentStoreTester {
 class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
   implicit val rand: Blake2b512Random = Blake2b512Random(Array.empty[Byte])
 
+  def checkData(
```

Comment:
I'd put those on the bottom so as not to intimidate the readers from the very header of the class :)

Reply:
Oh, I'm late to the party :D

Reply:
you said yourself that you want to have the methods defined before they are used :)

Reply:
Would approve anyway :D